### ISSUE TITLE
move test directory to be inside the docusign_esign directory

### ISSUE THREAD

**Issue by juliocarrera** [docusign/docusign-python-client#16]
This isn't really a bug. But when the docusign package is installed the test directory is placed at the same level as all the other site-packages in a Python installation. If other packages did the same thing and placed their test directories at the same level then I don't know what would happen but it's probably not good.

**Comment by mmallis87** [comment]
@juliocarrera good point. I created ticket DCM-2744 for us to look into removing test from the distribution, while keeping it in Github.

**Comment by garg-mudit** [comment]
Closing this issue as it has since been resolved. If the problem persists, feel free to reopen it. For any new or unrelated issues, please open a separate ticket.
